docs/Reference/Pantheon-API-Methods.md

```diff
@@ -904,6 +904,77 @@ Returns the account balance of the specified address.
     }    
     ```
     
+### eth_getProof
+
+Returns the account and storage values of the specified account, including the merkle proof.
+
+The API allows IoT devices or mobile apps which are unable to run light clients to verify responses from untrusted sources, by using a trusted block hash. 
+
+**Parameters**
+
+`DATA` - 20-byte address of the account or contract.
+
+`ARRAY` - 32-byte storage values of the account or contract address as described in [eth_getStorageAt](#eth_getstorageat)
```

Comment:
(a) `eth_getStorageAt` is a forward reference, so this is the first place it would be describe in the docs.
(b) `eth_getStorageAt` doesn't describe the storage address.  I have no idea why the ethereum JSON-RPC docs are referencing that RPC, but there is nothing there of relevance.

```suggestion
`ARRAY` - Array of 32-byte storage keys to generate proofs for.
```

Reply:
Thanks, will implement this.